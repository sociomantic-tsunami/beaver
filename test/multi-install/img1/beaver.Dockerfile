# Copyright dunnhumby Germany GmbH 2018.
# Distributed under the Boost Software License, Version 1.0.
# (See accompanying file LICENSE.txt or copy at
# http://www.boost.org/LICENSE_1_0.txt)
#
# This Dockerfile is just for testing
FROM sociomantictsunami/dlang:v2
RUN touch /BUILT-img1-Dockerfile
COPY img1/build /BUILT-img1-Dockerfile-COPY
