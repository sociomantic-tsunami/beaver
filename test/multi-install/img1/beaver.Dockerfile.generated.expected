# Copyright sociomantic labs GmbH 2018.
# Distributed under the Boost Software License, Version 1.0.
# (See accompanying file LICENSE.txt or copy at
# http://www.boost.org/LICENSE_1_0.txt)
#
# This Dockerfile is just for testing
FROM sociomantictsunami/dlang:xenial-v2
RUN touch /BUILT-img1-Dockerfile
COPY img1/build /BUILT-img1-Dockerfile-COPY
COPY img1/ /docker-tmp
RUN /docker-tmp/build && rm -fr /docker-tmp
